homeassistant/components/bt_smarthub/device_tracker.py

```diff
@@ -35,8 +39,12 @@ def __init__(self, config):
         """Initialise the scanner."""
         _LOGGER.debug("Initialising BT Smart Hub")
         self.host = config[CONF_HOST]
+        self.smarthub_model = config.get(CONF_SMARTHUB_MODEL)
         self.last_results = {}
         self.success_init = False
+        self.smarthub = BTSmartHub(
```

Comment:
Please instantiate the `BTSmartHub` inside `get_scanner` and pass the instance to `BTSmartHubScanner` when instantiating that.

Reply:
Please see my latest commit which addresses this request.